pusher-api/overview.md

```diff
@@ -6,10 +6,14 @@ With subscriptions, the Pusher API will immediately update you of those events t
 * [Plan subscriptions](#plan-subscriptions)
 * [Manage subscriptions](#manage-subscriptions)
 
+
 ## Understand how subscriptions work
 The Pusher API provides events for you to listen to certain activities of the Zettle Go app at a working HTTPS endpoint on your server.
 
-After you subscribe to events, when an event is triggered, the Pusher API sends a `POST` request that contains a `payload` field. The field contains event information to the HTTPS endpoint in real time.
+When an event that you have subscribed is triggered, the Pusher API sends the event in a `POST` request to the HTTPS endpoint. The HTTPS endpoint is used as the destination URL. 
+The request contains a `payload` field with event information in real time.
+ 
+ After the Pusher API sends a `POST` request, if the destination URL doesn't return a valid HTTP response within 10 seconds, the request times out.
 
```

Comment:
This is probably covered elsewhere in the documentation but perhaps mention that too many timeouts can lead to the subscription being deactivated?